source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -19,22 +19,20 @@ You can use the [discovery endpoint][external.oidc-discovery] to get information
 * information about the keys
 * supported scopes, which will contain the user attributes your service can request
 
-## Make an authorisation request
-To make an authorisation request, your application should first send your user to the authorisation URL.
+## Make a request to the `/authorize` endpoint
 
-The way you make an authorisation request will depend on whether you need:
+You can send a request to the `/authorize` endpoint:
 
-* an authentication request only
-* an authentication and identity assurance request (if you need identity, you must include authentication too)
+* to authenticate your user
+* to verify your user’s level of identity confidence - you must have authenticated your user first
```

Comment:
```suggestion
* authenticate your user
* verify your user’s level of identity confidence - you must have authenticated your user first
```

Reply:
Wondering about the word 'verify' - we tend to talk about identity checks, so maybe 'check' would work here. Still need to join this up across the docs though for consistency, so it could wait!

Reply:
Good point. I think we should change this here, and also record it on our restructuring document, to check that we are consistent.